### DISCUSSION TITLE
[Hunt] * | groupby "Insert field here" Request Failed code 500

### DISCUSSION THREAD

**Discussion by zaphode** [Security-Onion-Solutions/securityonion#3658] [Unsupported Versions] [answered]
I imported a syslog.evtx file and am trying to do a groupby command of the "process.executable" field to search for outside the norm processes. It seems like this should work. My first guess was that maybe it did not like the "\\" in file path for the executable but groupby 'user.name' works fine and the name format is  "TEST\someuser" so it does not seem to be the '\\'. 
 Here is the syntax I am using in Hunt query bar.  

`* | groupby process.executable`

**Comment by zaphode** [comment, marked as answer]
Ok it seems that this problem is more wide spread than just the "process.executable" grouping. I was trying to groupby uniqe alerts and that also threw the 500 error. Any idea what could be going wrong.

**Reply by jertel** [reply]
Look at your `/opt/so/log/soc/sensoroni-server.log` for more information about the error.

**Reply by zaphode** [reply]
> Look at your `/opt/so/log/soc/sensoroni-server.log` for more information about the error.

So I did this and it looks like I may be able to try the sort by setting the "fielddata=true on [rule.name]". I can't figure out exactly where to set this. What I am trying to do is groupby process.executable and get a count of unique rules by using groupby rule.name

![image](https://user-images.githubusercontent.com/3597404/113598478-ce4a2a00-9602-11eb-9253-fd97be1caacc.png)

It seems like this should be working. It shows up in Kibana as a Keyword field and the Index object has the unique count I am trying to groupby in Hunt. Is there something I am missing here? 

![image](https://user-images.githubusercontent.com/3597404/113602389-29cae680-9608-11eb-8d9b-30c284bd32e3.png)

**Reply by jertel** [reply]
Sounds like something is wrong with your index mappings. Check the Kibana index mappings and search for `rule.name`. It should look like this:
![image](https://user-images.githubusercontent.com/16639611/113721943-f3e83980-96bd-11eb-8000-8fcb13caf15f.png)

**Reply by zaphode** [reply]
This does work in Hunt in the full version of SO I am running. It seems to only be broken in the import install.  

That looks how it should. 
![image](https://user-images.githubusercontent.com/3597404/113728757-f9dd1b00-96bb-11eb-93cf-7727e302b3bb.png)

**Reply by jertel** [reply]
I just started up an import install and the groupby is working, so it appears to be something specific with your environment. Check your sensoroni-server log for a "Fetching Field" line. Do you have an error following that line?

**Reply by zaphode** [reply]
There were no lines in the Sensoroni log containing "Fetching Field"

I did a fresh install and it seems to be working now so I don't think we need to track it any farther unless someone else runs into it.
